src/components/HomepageFeatures/index.js

```diff
@@ -8,7 +8,7 @@ const FeatureList = [
     Svg: require("@site/static/img/undraw_digital_currency_qpak.svg").default,
     description: (
       <>
-        A simple open framework to deliver best practice community banking and
+        An open framework to deliver best practice federated applications and
```

Comment:
Just want to add I really like this, obviously I was coming at this from the banking payment angle, but definitely see this now after spending a little more timein the project.